api/hearings/data/hearingRequests.mock.data.ts

```diff
@@ -150,7 +148,6 @@ export const HEARING_REQUEST_RESULTS: HearingRequestMainModel[] = [
             partyName: 'Jane and Smith',
             partyType: PartyType.IND,
             partyRole: 'appellant',
-            partyChannel: 'inPerson',
```

Comment:
in this mock data, after the partyChannel: 'inPerson', is removed here, could this be added to the the individualDetails.preferredChannel.